README.md

```diff
@@ -1,29 +1,64 @@
 # PkgAuthentication
 
-Authentication to private Julia package servers
+Interactive browser-based authentication to private Julia Pkg servers.
 
-```
-    authenticate(pkgserver)
+## Setup
+
+#### Step 1: Make sure that PkgAuthentication.jl is installed in the default global Julia package environment (`v1.x`)
+
+```julia
+julia> delete!(ENV, "JULIA_PKG_SERVER")
+
+julia> import Pkg
 
-Starts browser based pkg-server authentication (blocking).
+julia> Pkg.activate("v$(VERSION.major).$(VERSION.minor)"; shared = true)
 
-`pkgserver` must be a URL pointing to a server that provides the `/pkgserver/challenge`,
-`/pkgserver/response`, and `/pkgserver/claimtoken` endpoints.
+julia> Pkg.add("PkgAuthentication")
 ```
 
-## Installation
-1. Make sure PkgAuthentication.jl is part of the default Julia environment.
-2. Put the following into your `startup.jl` to enable authentication for server as configured in JULIA_PKG_SERVER environment variable.
+#### Step 2: Put the following snippet into your [`startup.jl`](https://docs.julialang.org/en/v1/manual/getting-started/) file
+
 ```julia
 # create a new anonymous module for the init code to not pollute the global namespace
 Base.eval(Module(), quote
-    using PkgAuthentication
-    PkgAuthentication.install()
+    ENV["JULIA_PKG_SERVER"] = "my-pkg-server.example.com"
```

Comment:
Wanna factor this out into a separate step? In many cases (for some value of "many", of course) this environment variable will already be set, in which case the user can't just copy that snippet into their startup.jl.

Reply:
Done!